shell/platform/darwin/ios/framework/Headers/FlutterBinaryMessenger.h

```diff
@@ -39,24 +49,24 @@ FLUTTER_EXPORT
  no reply.
 
  - Parameters:
+   - channel: The channel name.
    - message: The message.
-   - channelName: The channel name.
  */
-- (void)sendBinaryMessage:(NSData* _Nullable)message
-              channelName:(NSString*)channelName;
+- (void)sendOnChannel:(NSString*)channel
```

Comment:
Nit: `sendMessage:onChannel:` reads more naturally to me, but I think is OK too.